src/sentry/static/sentry/app/views/releases/list/index.jsx

```diff
@@ -60,9 +60,11 @@ class OrganizationReleases extends AsyncView {
     const targetQueryParams = {};
     if (query !== '') {
       targetQueryParams.query = query;
+      targetQueryParams.project = this.props.location.query.project;
```

Comment:
I'd imagine we should keep all of the parameters from our global selection bar when performing a search.  We have a utility function `extractSelectionParameters` that will do this. It keeps project, env, and time period if any of them are set.

Reply:
Okay, cool. I did think about just using location.query but didn't want to consider all of the possible side effects and wanted to keep the change minimal to avoid introducing new bugs.

I'll definitely use this function though - it's the better way.